lib/commands/import/command.js

```diff
@@ -0,0 +1,39 @@
+"use strict";
+
+const UI = require('../../ui').UI;
+const CLIOptions = require('../../cli-options').CLIOptions;
+const Container = require('aurelia-dependency-injection').Container;
+const Analyzer = require('../../import/analyzer');
+const ImportEngine = require('../../import/import-engine');
+const ImportDefault = require('../../import/import-default');
+
+module.exports = class {
+
+  static inject() { return [Container, UI, CLIOptions]; };
+
+  constructor(container, ui, options) {
+    this.container = container;
+    this.ui = ui;
+    this.options = options;
+  }
+
+  execute(args) {
+    let analyzer = this.container.get(Analyzer);
+    return analyzer
+      .setPackage(args[0])
+      // analyze contextual information (CLI parameters, given package)
+      .execute()
+    //*/
+      // configure aurelia.json, install custom tasks, run additional scripts
+      .then(() => {
+        let engine = new ImportEngine(
+          analyzer.result.project,
```

Comment:
Is there some reason why the result isn't passed through the promise instead of making it a property on the analyzer instance?